docs/data.md

```diff
@@ -151,3 +151,128 @@ i.e., push the computation region settings high in the code structure.
 However, for running a job, the computational region should be somewhere
 otherwise it will rely on whatever is the (last set) computational region in the
 current mapset.
+
+## Using local scratch space for temporary data
+
+### Running a single job
+If you are processing a large number of temporary files with GRASS GIS, it can be useful to write the   
```

Comment:
Just because GRASS itself has temporary files which users sometimes manage, I would change

> temporary files with GRASS GIS

to

> temporary files with GRASS GIS (e.g., temporary rasters or mapsets)

Or talk about data rather than files, but that may require more changes.